src/app/adopt/page.tsx

```diff
@@ -0,0 +1,38 @@
+import Container from "@/components/Container";
+import PetThumbnails from "@/components/PetThumbnails";
+import TermsAndConditions from "@/components/TermsAndConditions";
+import MainLayout from "@/layouts/MainLayout";
+import { Icon } from "@iconify/react/dist/iconify.js";
+import { Link } from "react-router-dom";
+
+interface AdoptionPageProps {
+  petName: string;
+}
+
+const AdoptionPage = ({ petName = "ไรลีย์" }: AdoptionPageProps) => {
+  return (
+    <div>
+      <Container className="flex flex-row items-center justify-between text-primary">
+        <Link to="/pets">
+          <Icon icon={"ion:chevron-back"} className="h-8 w-8" />
+        </Link>
+        <div className="visible flex flex-row justify-between text-4xl font-bold underline  md:hidden">
+          {petName}
+        </div>
+      </Container>
+      <Container className="flex flex-col items-center gap-6 py-10 md:flex-row md:items-start md:justify-between">
```

Comment:
Change `gap-6` to `gap-8` (เดิมใน figma เป็น Auto แต่เอาเป็น gap-8 ดีกว่า จะได้เหมือนในหน้า admin)